adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegral.m

```diff
@@ -55,39 +109,51 @@ + (GADVersionNumber)adapterVersion {
     version.majorVersion = components[0].integerValue;
     version.minorVersion = components[1].integerValue;
     version.patchVersion = components[2].integerValue * 100 + components[3].integerValue;
-    if (components.count >= 5) {
-      version.patchVersion = version.patchVersion * 100 + components[4].integerValue;
-    }
   }
   return version;
 }
 
-+ (nullable Class<GADAdNetworkExtras>)networkExtrasClass {
-  // TODO: Return the class for passng in mediation extras (if any). Else, return `Nil`.
-  return Nil;
+- (void)collectSignalsForRequestParameters:(nonnull GADRTBRequestParameters *)params
+                         completionHandler:(nonnull GADRTBSignalCompletionHandler)completionHandler {
+    if (completionHandler) {
+        completionHandler([MTGBiddingSDK buyerUID],nil);
+    }
 }
 
-- (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
-                   completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+- (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
```

Comment:
@Mintegral-Lee 

Question: For all ad formats, will Mintegral SDK return a failure callback if trying to load ads when it's not initialized yet?

Reply:
Yes, there will be a load failure message returned.